composer.json

```diff
@@ -140,12 +136,21 @@
         {
             "type": "composer",
             "url": "https://asset-packagist.org"
+        },
+        {
+            "type": "vcs",
+            "url": "https://git.drupalcode.org/issue/jsonapi_cross_bundles-3172761"
+        },
+        {
+            "type": "vcs",
+            "url": "https://git.drupalcode.org/issue/theme_switcher-3154752"
```

Comment:
It's not ideal to have to include these repos, but both the changes should eventually be merged into the main branch and released.  So we can remove these after that has happened.